ibis/tests/test_typing.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from typing import Any
+
+from ibis.common.typing import get_type_hints
+
+
+class TestClass:
+    a: int
+
+    @property
+    def b(self) -> str:
+        return "test"
+
+    @property
+    def c(self) -> Any:
+        raise AttributeError("Property 'c' raises AttributeError")
+
+
+def test_get_type_hints_with_attribute_error():
+    hints = get_type_hints(TestClass, include_properties=True)
+    assert hints == {"a": int, "b": str}
```

Comment:
This is failing because the attribute error isn't raised until accessing an instance property, but the function that extracts these properties operates on classes, not instances.